src/components/SwapForm/SwapModal/SwapModalAreYouSure.tsx

```diff
@@ -0,0 +1,137 @@
+import { Currency, CurrencyAmount } from '@kyberswap/ks-sdk-core'
+import { Trans } from '@lingui/macro'
+import React, { Dispatch, SetStateAction, useState } from 'react'
+import { X } from 'react-feather'
+import { Flex, Text } from 'rebass'
+import styled from 'styled-components'
+
+import { ButtonErrorStyle, ButtonOutlined } from 'components/Button'
+import Modal from 'components/Modal'
+
+const ModalContentWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  padding: 24px 24px 28px;
+  background-color: ${({ theme }) => theme.tableHeader};
+`
+
+const StyledInput = styled.input`
+  margin-top: 24px;
+  background: ${({ theme }) => theme.buttonBlack};
+  border-radius: 999px;
+  padding: 8px 16px;
+  font-size: 16px;
+  outline: none;
+  color: ${({ theme }) => theme.text};
+  border: none;
+  &::placeholder {
+    color: ${({ theme }) => theme.disableText};
+  }
+`
+
+const StyledCloseIcon = styled(X)`
+  height: 28px;
+  width: 28px;
+  :hover {
+    cursor: pointer;
+  }
+
+  > * {
```

Comment:
`color: theme.text` is not working?

Reply:
Tbh, I don't know why I put it here. But it's work.

Reply:
pls update color: theme.text